cdap-ui/app/features/flows/controllers/tabs/runs/flowlets/output-ctrl.js

```diff
@@ -39,67 +43,66 @@ angular.module(PKG.name + '.feature.flows')
               if (res.series[0]) {
                 updateOutput(res.series[0].data);
               } else {
-                  var val = [];
+                var val = [];
 
-                  for (var i = 60; i > 0; i--) {
-                    val.push({
-                      time: Math.floor((new Date()).getTime()/1000 - (i)),
-                      y: 0
-                    });
-                  }
+                for (var i = 60; i > 0; i--) {
```

Comment:
Except for this ```for``` loop the ```updateOutput``` function and this function are the same right? Is it possible to have some code re-use here?